full/coupler_main.F90

```diff
@@ -333,253 +333,14 @@ program coupler_main
-  logical :: do_debug=.FALSE.       !< If .TRUE. print additional debugging messages.
-  integer :: check_stocks = 0 ! -1: never 0: at end of run only n>0: every n coupled steps
-  logical :: use_hyper_thread = .false.
-
-  namelist /coupler_nml/ current_date, calendar, force_date_from_namelist,         &
-                         months, days, hours, minutes, seconds, dt_cpld, dt_atmos, &
-                         do_atmos, do_land, do_ice, do_ocean, do_flux,             &
-                         atmos_npes, ocean_npes, ice_npes, land_npes,              &
-                         atmos_nthreads, ocean_nthreads, radiation_nthreads,       &
-                         concurrent, do_concurrent_radiation, use_lag_fluxes,      &
-                         check_stocks, restart_interval, do_debug, do_chksum,      &
-                         use_hyper_thread, concurrent_ice, slow_ice_with_ocean,    &
-                         do_endpoint_chksum, combined_ice_and_ocean
-
-  integer :: initClock, mainClock, termClock
-
-  integer :: newClock0, newClock1, newClock2, newClock3, newClock4, newClock5, newClock7
-  integer :: newClock6f, newClock6s, newClock6e, newClock10f, newClock10s, newClock10e
-  integer :: newClock8, newClock9, newClock11, newClock12, newClock13, newClock14, newClocka
-  integer :: newClockb, newClockc, newClockd, newClocke, newClockf, newClockg, newClockh, newClocki
-  integer :: newClockj, newClockk, newClockl
-
-  integer :: id_atmos_model_init, id_land_model_init, id_ice_model_init
-  integer :: id_ocean_model_init, id_flux_exchange_init
-
-  character(len=80) :: text
-  character(len=48), parameter                    :: mod_name = 'coupler_main_mod'
-
-  integer :: outunit
-  integer :: ensemble_id = 1
-  integer, allocatable :: ensemble_pelist(:, :)
-  integer, allocatable :: slow_ice_ocean_pelist(:)
-  integer :: conc_nthreads = 1
-  real :: dsec, omp_sec(2)=0.0, imb_sec(2)=0.0
-
-!#######################################################################
-  INTEGER :: i, status, arg_count
-  CHARACTER(len=256) :: executable_name, arg, fredb_id
-
-#ifdef FREDB_ID
-#define xstr(s) str(s)
-#define str(s) #s
-  fredb_id = xstr(FREDB_ID)
-#else
-#warning "FREDB_ID not defined. Continuing as normal."
-  fredb_id = 'FREDB_ID was not defined (e.g. -DFREDB_ID=...) during preprocessing'
-#endif
-
-  arg_count = command_argument_count()
-  DO i=0, arg_count
-     CALL get_command_argument(i, arg, status=status)
-     if (status .ne. 0) then
-        write (error_unit,*) 'get_command_argument failed: status = ', status, ' arg = ', i
-        stop 1
-     end if
-
-     if (i .eq. 0) then
-       executable_name = arg
-     else if (arg == '--fredb_id') then
-        write (output_unit,*) TRIM(fredb_id)
-        stop
-     end if
-  END DO
-
-  if (arg_count .ge. 1) then
-    write (error_unit,*) 'Usage: '//TRIM(executable_name)//' [--fredb_id]'
-    stop 1
-  end if
```

Comment:
You need to add this to the description of this PR or you need to open a new PR that has a more general cleanup/removal of parts of the code

Reply:
Added back in and ready for another round of review!